climpred/utils.py

```diff
@@ -76,6 +162,23 @@ def get_comparison_function(comparison, list_):
         is_in_list(comparison, list_, 'comparison')
         return getattr(comparisons, '_' + comparison)
 
+      
+def get_lead_pdoffset_args(units, lead):
+
+    lead_units_list = ['years', 'months', 'weeks', 'days']
+
+    if units in lead_units_list:
+        offset_args_dict = {units: lead}
+    elif units == 'seasons':
+        offset_args_dict = {'months': lead + 3}
+    elif units == 'pentads':
+        offset_args_dict = {'days': lead + 5}
+    else:
+        offset_args_dict = {units: lead}
+        print(units, ' is not a valid choice')
```

Comment:
Should this throw a breaking error if it isn't in our expected units?

Reply:
We should never get to that else statement since we previously checked that the object has valid units when it was created, but I couldn't come up with anything better to do in the else.   Suggestions?